emoji.go

```diff
@@ -99,7 +137,7 @@ func Println(a ...interface{}) (int, error) {
 
 // Printf is fmt.Printf which supports emoji
 func Printf(format string, a ...interface{}) (int, error) {
-	return fmt.Printf(compile(fmt.Sprintf(format, a...)))
+	return fmt.Print(compile(fmt.Sprintf(format, a...)))
```

Comment:
Is there a reason for this change?

Reply:
Changed from the output of https://github.com/golangci/golangci-lint

```
$ golangci-lint run .
emoji_test.go:189:22: S1019: should use make([]byte, buff.Len()) instead (gosimple)
                bc := make([]byte, buff.Len(), buff.Len())
                                   ^
emoji.go:140:20: SA1006: printf-style function with dynamic format string and no further arguments should use print-style function instead (staticcheck)
        return fmt.Printf(compile(fmt.Sprintf(format, a...)))
                          ^
```

Happy to move to a separate PR or remove if you want!

Reply:
I see . Thank you, I understand 👍